src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/InnvilgeBarnetilsynV2.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from 'react';
+
+import { BodyShort, ErrorMessage, HStack, Link, VStack } from '@navikt/ds-react';
+
+import Beregningsresultat from './Beregningsresultat';
+import { Vedtaksperioder } from './Vedtaksperioder';
+import { useApp } from '../../../../../context/AppContext';
+import { useBehandling } from '../../../../../context/BehandlingContext';
+import { useSteg } from '../../../../../context/StegContext';
+import { FormErrors, isValid } from '../../../../../hooks/felles/useFormState';
+import DataViewer from '../../../../../komponenter/DataViewer';
+import SmallButton from '../../../../../komponenter/Knapper/SmallButton';
+import Panel from '../../../../../komponenter/Panel/Panel';
+import { StegKnapp } from '../../../../../komponenter/Stegflyt/StegKnapp';
+import { Steg } from '../../../../../typer/behandling/steg';
+import { byggHenterRessurs, byggTomRessurs, RessursStatus } from '../../../../../typer/ressurs';
+import {
+    BeregningsresultatTilsynBarn,
+    InnvilgeBarnetilsynRequestV2,
+    InnvilgelseBarnetilsyn,
+    validerVedtaksperioder,
+    VedtaksperiodeTilsynBarn,
+    vedtaksperiodeTilVedtakperiodeTilsynBarn,
+} from '../../../../../typer/vedtak/vedtakTilsynBarn';
+import { FanePath } from '../../../faner';
+import { lenkerBeregningTilsynBarn } from '../../../lenker';
+import { initialiserVedtaksperioder } from '../VedtakBarnetilsynUtils';
+
+interface Props {
+    lagretVedtak?: InnvilgelseBarnetilsyn;
+    vedtaksperioderForrigeBehandling?: VedtaksperiodeTilsynBarn[];
+}
+
+export const HeadingBeregning: React.FC = () => {
+    return (
+        <HStack gap="4" align={'end'}>
+            {lenkerBeregningTilsynBarn.map((lenke, indeks) => (
+                <BodyShort key={indeks} size={'small'}>
+                    <Link key={indeks} href={lenke.url} target="_blank" variant="neutral">
+                        {lenke.tekst}
+                    </Link>
+                </BodyShort>
+            ))}
+        </HStack>
+    );
+};
+
+export const InnvilgeBarnetilsynV2: React.FC<Props> = ({
```

Comment:
Jeg lurer litt på om det hadde vært mulig å legge inn den manglende funksjonaliteten i eksisterende komponent, men at den ikke vises hvis ikke feature-togglet
Eller hadde det blitt for rotete?

Reply:
Tror det kan blir rotete. Jeg liker bedre å ha to komponenter også heller ta oppryddigsjobben med å fjerne det gamle når vi ser at det nye funker.